src/test/scala/edu/ie3/simona/model/grid/Transformer3wModelSpec.scala

```diff
@@ -251,16 +237,10 @@ class Transformer3wModelSpec
           transformerTappingModel shouldBe expectedTappingModel
           amount shouldBe transformer3wInput.getParallelDevices
           powerFlowCase shouldBe PowerFlowCaseC
-          r should equalWithTolerance(
-            Quantities.getQuantity(3.185595567e-6, PU),
-            1e-8
-          )
-          x should equalWithTolerance(
-            Quantities.getQuantity(556.0941828e-6, PU),
-            1e-8
-          )
-          g shouldBe Quantities.getQuantity(0d, PU)
-          b shouldBe Quantities.getQuantity(0d, PU)
+          r =~ Each(3.185595567e-6)
```

Comment:
@sebastian-peter could you have a look here, why this is not identified as false, while `Math.abs(r.toEach - 3.185595567e-6) < 1e-8 shouldBe true` works as expected?

Reply:
Edit: With the fix within Transformer3wTestData it is now necessary to change the value (e.g. 1.0e-1) to have same behaviour (in the meaning of result is wrong but test does not identify this)